.github/workflows/build-bootc.yaml

```diff
@@ -6,19 +6,22 @@ on:
 
 env:
   REGISTRY: quay.io
-  REPOSITORY: flightctl
 
 jobs:
-  build-and-push-centos:
+  build-and-push:
     runs-on: ubuntu-latest
+  
+    strategy:
+      matrix:
+        flavor: [centos, fedora, rhel]
```

Comment:
Hrm sorry if I'm confused...why are we building/ pushing RHEL here when we have the separate workflow for doing just that in [build-rhel-bootc.yaml](https://github.com/flightctl/flightctl-demos/pull/18/files#diff-8b55cf759587f91d7fbc491ebda955868bdb37b7b3eb7591d6c54fd0bbb407f1)

Reply:
Works for me! QQ though for @oglok If we're now building the RHEL bootc image here do we need to to have the manually triggered workflow as well for doing the same thing?

Reply:
yes, my intention is that build-bootc.yaml will only build the bootstrap images for the three variants, and the other one is just an experimental pipeline to cover the end to end.